yarn.lock

```diff
@@ -7738,7 +7738,7 @@ __metadata:
     "@babel/preset-typescript": ^7.26.0
     "@hypothesis/frontend-build": ^3.0.0
     "@hypothesis/frontend-shared": ^9.0.0
-    "@hypothesis/frontend-testing": ^1.3.1
+    "@hypothesis/frontend-testing": ^1.5.0
```

Comment:
I took the opportunity to update this, to get IDE auto-completion on the types it exposes in most recent versions.